R/pandoc.R

```diff
@@ -551,25 +551,34 @@ unix_mathjax_path <- function() {
 
 
 pandoc_html_highlight_args <- function(template,
```

Comment:
I think this function would be marginally more clear if `highlight` was the first argument because is `template` only used when `highlight` is "default" (and then you could make it `template = "default"`)